.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "@hermeznetwork/eslint-config-hermez"
```

Comment:
This setting is in package.json, if you prefer this way instead the other, you should remove it from there